### ISSUE TITLE
Implement CRUD methods for files and directories in a FAT-formatted partition

### ISSUE THREAD

**Issue by basvandewiel** [basvandewiel/doscontainer#21]
*What*
Implement methods for CRUD operations on files and directories in a FAT-formatted partition.

*Why*
Only IO.SYS is a special case that must be written to a specific location. Any and all other files are just "regular" files that can be located anywhere. The base OS already needs MSDOS.SYS and COMMAND.COM, so that's two files that can be written in a generic way. If we're going to support that anyway, we can just as well expand our capabilities into full CRUD and circumvent requirements on the host OS for the image build pipeline.

In short: by integrating CRUD here, we forego the need to mount/unmount our FAT16 partitions from image files as writable on the host-OS, which may introduce all manner of issues.

This is a nice-to-have feature, though so it's not part of the minimally viable milestone.
